### ISSUE TITLE
Keywords which contain inline arguments are not highlited when using defined variables

### ISSUE THREAD

**Issue by himperion92** [nokia/RED#187]
I'm using Eclipse Oxygen v2.0 and RED v0.8.3. The problem comes when you define a keyword which makes use of inline arguments, for instance: 
```
*** Keywords ***
Navigate to url ${input_url}
    Selenium2Library.Go to    ${input_url}
```

Then, when you call this keyword in a test case using a defined variable as an argument, only the variable is highlighted but not the keyword.

```
*** Variables ***
${MAIN_URL}    https://www.phptravels.net

*** Test Cases ***
Check successful login with correct credentials 
    Open Firefox Explorer with url ${LOGIN_URL}
```
Here is a screenshot with this example: 
![image](https://user-images.githubusercontent.com/33331691/36787886-11406b4c-1c8c-11e8-8f8c-b4bbd0f4ccd2.png)

**Comment by KrzysztofJozefowicz** [comment]
confirmed, in such case KW should be colored as "keyword" blue/brown when having variable.

**Comment by michalanglart** [comment]
fixed in 0.8.5
